backend/capellacollab/projects/toolmodels/modelsources/git/models.py

```diff
@@ -48,16 +50,18 @@ class Config:
 
 class DatabaseGitModel(Base):
     __tablename__ = "git_models"
-    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
-    name = Column(String)
-    path = Column(String)
-    entrypoint = Column(String)
-    revision = Column(String)
-    primary = Column(Boolean)
-    model_id = Column(Integer, ForeignKey("models.id"))
-    model = relationship("DatabaseCapellaModel", back_populates="git_models")
-    username = Column(String)
-    password = Column(String)
+    id: int = Column(Integer, primary_key=True, index=True, autoincrement=True)
+    name: str = Column(String)
+    path: str = Column(String)
+    entrypoint: str = Column(String)
+    revision: str = Column(String)
+    primary: bool = Column(Boolean)
+    model_id: int = Column(Integer, ForeignKey("models.id"))
+    model: "toolmodels_models.DatabaseCapellaModel" = orm.relationship(
+        "DatabaseCapellaModel", back_populates="git_models"
+    )
+    username: str = Column(String)
+    password: str = Column(String)
```

Comment:
Yes, totally agree that the issue is in most cases not the type annotation but rather the missing `nullable=False`. But since this is a general problem and not specific to an issue in this PR I created #572 and we can keep it like this here.